earth2mip/ensemble_utils.py

```diff
@@ -150,6 +150,19 @@ def to(self, *args, **kwargs):
         return self
 
 
+# TODO change all noise funcs to update x and not return noise and unify functions
+def generate_model_noise_correlated(x,
+                                    time_step,
+                                    reddening,
+                                    device,
+                                    noise_injection_amplitude,
+                                    ):
+    shape = x.shape
+    dt = torch.tensor(time_step.total_seconds()) / 3600.0
+    noise = noise_injection_amplitude * dt * brown_noise(shape, reddening).to(device)
+    return x * (1.0 + noise)
```

Comment:
This function can be tightened up:

```suggestion
def add_multiplicative_noise(x, amplitude, reddening):
    noise = amplitude * brown_noise(shape, reddening).to(x.device)
    return x * (1.0 + noise)
    
    
dt =  torch.tensor(time_step.total_seconds()) / 3600.0
x_perturbed = add_multiplicative_noise(x, dt * noise_injection_amplitude)
```

Reply:
from 5 args to 2.